modules/apps/portal-template/portal-template-velocity/src/main/java/com/liferay/portal/template/velocity/internal/VelocityManager.java

```diff
@@ -111,8 +116,9 @@ public void init() throws TemplateException {
 			extendedProperties.setProperty(
 				StringBundler.concat(
 					"liferay.", VelocityEngine.RESOURCE_LOADER, ".",
-					VelocityTemplateResourceLoader.class.getName()),
-				_templateResourceLoader);
+					VelocityManager.VelocityTemplateResourceLoader.class.
```

Comment:
for this case we can make the inner class public since it is used in the production code.